modules/python/test/test_dnn.py

```diff
@@ -148,7 +137,7 @@ def test_blobFromImage(self):
 
     def test_face_detection(self):
         testdata_required = bool(os.environ.get('OPENCV_DNN_TEST_REQUIRE_TESTDATA', False))
-        proto = self.find_dnn_file('dnn/opencv_face_detector.prototxt2', required=testdata_required)
+        proto = self.find_dnn_file('dnn/opencv_face_detector.prototxt', required=testdata_required)
```

Comment:
Sure, thanks! I think we'll merge this PR soon when I fix other comments. But yeah, there are tests which just are skipped because of unfound models (i.e. ONNX' shufflenet). I'll open a PR with striter tests.